common/utils/src/main/resources/error/error-classes.json

```diff
@@ -914,6 +914,11 @@
           "The <exprName> must be between <valueRange> (current value = <currentValue>)."
         ]
       },
+      "VARIANT_TYPE" : {
```

Comment:
I think `UNSUPPORTED_VARIANT_TYPE` may be a better name.

Reply:
Addressed in another comment. Resolving